hack/release.sh

```diff
@@ -8,10 +8,11 @@ BUILDDIR=$(realpath $DIR)/build
 
 # Ensure we're on the master branch
 check_branch() {
+    version=$1
     curr_branch=$(git rev-parse --abbrev-ref HEAD)
-    if [ $curr_branch != "master" ]
+    if [ $curr_branch != "v${version}" ]
```

Comment:
Can you elaborate on this a bit?  Why is this better than releasing from master?

Reply:
If we release from master branch, we need to merge changes to docs, charts and things related to fission version into master branch first. And then add changelog.md in a separate commit.
If we release from a branch other than the master (i.e v0.5.0), we can commit changes in that branch first. And after some final check (or review) we can merge these changes into master with only one commits rather than multiple ones.

Reply:
Why can't we add the changelog in the same commit? Does the changelog generator depend on there being a version branch?